lib/email/index.ts

```
@@ -0,0 +1,65 @@
+    const template = getTeamInviteEmailTemplate(props);
+    await this.sendEmail(to, template.subject, template.html);
+  }
+
+  public static async sendPasswordResetEmail(
+    to: string,
+    props: ResetPasswordEmailProps
+  ) {
+    const template = getResetPasswordEmailTemplate(props);
+    await this.sendEmail(to, template.subject, template.html);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)